src/databaseSvc/databaseManipulation.py

```diff
@@ -60,19 +60,16 @@ def replace_event(self, event_id: str, new_event: dict) -> bool:
 
     @Logger()
     def delete_event(self, event_id: str) -> bool:
-        return self.session.delete_one({'Event_id': event_id}).deleted_count == 1
+        return self.session.delete_one({'Event_id': event_id}).deleted_count

```

Comment:
We expect bool as a result but `deleted_count` is int. Also what if it somehow delete more than one event?

Reply:
Right. Do we have any ways to check, what exactly thing was deleted from database (from mongo cursor or smth, not force compairing). We can return this from method.